qiskit/transpiler/passes/layout/vf2_layout.py

```diff
@@ -128,6 +128,15 @@ def run(self, dag):
             cm_graph_edge_count = len(self.coupling_map.graph.edge_list())
             self.max_trials = max(im_graph_edge_count, cm_graph_edge_count) + 15
 
+        # If call_limit is not set, scale call_limit based the size of
+        # `im_graph`  using the exponential relationship
+        #    call_limit = A*10^{B*|im_graph|}.
+        # See Issue #7705 for the development of the A,B coefficients.
+        if self.call_limit is None:
+            A = 9.12e-4
+            B = 0.354
+            self.call_limit = A * 10**(B*len(im_graph.edge_list()))
```

Comment:
I think this will need to be case as an `int`, I think retworkx might error if we pass a float to the `call_limit` argument on `vf2_mapping()`.

Reply:
Resolved by https://github.com/HM0880/qiskit-terra/commit/f06e383f56507cd9e2f54bc4d686533603f3311e